scripts/gen-c

```diff
@@ -14,21 +21,39 @@ EOF
   exit 1
 }
 
-test -x "$tree_sitter" || error "missing executable $tree_sitter"
-
 # Generate C source code for the grammar, in two passes:
 #
-# 1. Generate the original grammar.json by executing grammar.js.
-# 2. Simplify grammar.json so it's compatible with ocaml-tree-sitter.
-# 3. Generate the C parser.
+# 1. Import files from semgrep-grammars
+# 1. Simplify grammar.json so it's compatible with ocaml-tree-sitter.
```

Comment:
should this be #2?

Reply:
oops, sorry about that. I rushed this PR as it was Friday night...